client/src/components/Footer/style.js

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+
+export const Container = styled('div')`
+  padding: ${({ theme: { spacings, constants } }) =>
+    `${spacings[3]} ${constants.general.paddingLeft}`};
+  ${({ theme }) => theme.media.mobile} {
+    padding: ${({ theme }) => `${theme.spacings[3]} ${theme.spacings[5]}`};
+  }
+  width: 100%;
+  min-height: 110px;
```

Comment:
add this to theme > constants > layout > footer > height, so all layout dimentions in the same place 
```suggestion
  min-height:  ${({ theme }) => theme.constants .layout.layout.footer.height};
```